pynecone/components/__init__.py

```diff
@@ -41,6 +41,11 @@
 stat_help_text = StatHelpText.create
 stat_label = StatLabel.create
 stat_number = StatNumber.create
+tag=Tag.create
+taglabel=TagLabel.create
```

Comment:
Can we use underscores for the multi-word ones, i.e. `tag_label`, `tag_left_icon`

Reply:
Sure, i will make the changes